prog/vm/nexus.rb

```diff
@@ -403,15 +403,13 @@ def write_params_json
         raise unless /Failed to stop .* Unit .* not loaded\./.match?(ex.stderr)
       end
 
-      host.sshable.cmd("sudo host/bin/setup-vm delete #{q_vm}")
+      # If there is a load balancer setup, we want to keep the network setup in
+      # tact for a while
+      action = vm.load_balancers.empty? ? "delete" : "delete_wo_net"
```

Comment:
maybe `delete_keep_net`? I presume that's what you mean here? like, delete, but *w*ith*o*ut deleting net?